plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/flex/CreateAppEngineFlexWtpProject.java

```diff
@@ -90,40 +88,24 @@ public String getDescription() {
   @Override
   public IFile createAndConfigureProjectContent(IProject newProject, AppEngineProjectConfig config,
       IProgressMonitor monitor) throws CoreException {
-    SubMonitor subMonitor = SubMonitor.convert(monitor, 100);
     IFile mostImportantFile =  CodeTemplates.materializeAppEngineFlexFiles(newProject, config,
-        subMonitor.newChild(30));
-    configureFacets(newProject, subMonitor.newChild(20));
-    if (!config.getUseMaven()) {
-      addDependenciesToProject(newProject, subMonitor.newChild(50));
-    }
+        monitor);
     return mostImportantFile;
   }
 
-  /**
-   * Add Java 8 and Dynamic Web Module facet
-   */
-  private void configureFacets(IProject project, IProgressMonitor monitor) throws CoreException {
```

Comment:
This is exactly what `AppEngineFlexWarFacet.installAppEngineFacet()` above does. No need to duplicate.